pkg/model/dev.go

```diff
@@ -202,6 +203,12 @@ type PersistentVolumeInfo struct {
 	Size         string `json:"size,omitempty" yaml:"size,omitempty"`
 }
 
+// InitContainer represents the initial container
+type InitContainer struct {
```

Comment:
Would it possible to include this config in the `okteto doctor` file?

Reply:
It is already done, we include the okteto manifest